Formula/vips.rb

```diff
@@ -11,20 +11,21 @@ class Vips < Formula
   end
 
   bottle do
-    sha256 arm64_big_sur: "96192c9244f233b59b24e56bdb5a52ffaae7ad79f35792b512c5956ceadb159a"
-    sha256 big_sur:       "1ff65470a885cb982a651a9aeb76be2188803f58071547a5c677718cb24ab889"
-    sha256 catalina:      "45e3e0ae7564b37ba1398b48465811889b42783e02b6c3491c0d3dabdda9e3ef"
-    sha256 mojave:        "b4aecdd43904e1acee4da8af57da53ebaa5e29f857c10c528226888457aa9a49"
+    sha256 arm64_big_sur: "7465e3684f04ad85463694971862aa3694ec67655b01e8d5d6eef6af22852af9"
+    sha256 big_sur:       "423bdd4aa945fefc6dd01646892c429494620cc942a96f25e2eab72cc8088df0"
+    sha256 catalina:      "8d60363727bb9f48e8733b8d570732734ae727809489547e8ec3c956a72569f2"
+    sha256 mojave:        "49429e971e5550fd703dcdcd3647a0d14c08267fa7e5c2ac4f6e0a402e92fb2f"
```

Comment:
Please don't touch the bottle block. (Where did these new checksums even come from?)

Reply:
Ooop, sorry. I didn't touch them, it must have been some strange accident. I'll remove this.

Reply:
Ah, these are the bottle checksums from 34f59d481e1b7a175b3ddb2a486c03865481d5fe. You must've started from an outdated branch when you created your PR. Do 
```
git fetch origin && git checkout -b <branch name> origin/master
```
to make sure your PR branch is up-to-date before working on changes.

Or, you can use `brew bump-formula-pr` (and amend the commits it creates automatically) and it should just do the right thing.